profile/legacy_profile.go

```diff
@@ -865,7 +865,6 @@ func parseThread(b []byte) (*Profile, error) {
 	// Recognize each thread and populate profile samples.
 	for !isMemoryMapSentinel(line) {
 		if strings.HasPrefix(line, "---- no stack trace for") {
-			line = ""
```

Comment:
Is this intentional?

Reply:
Yes, it fixed a lint warning that this assignment is redundant - we break out of the loop and this var is not used after the loop.

Reply:
Ah, okay. Thanks.